### PR TITLE
Add an ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by ficristo**
I'm tired to write the same things on new issues so I've written this, heavily inspired by [Atom](https://github.com/atom/atom/blob/ad24aaef3565600313d018995e2451277b4cefc2/ISSUE_TEMPLATE.md)

### CODE REVIEW

**Comment by redmunds** [comment]
@ficristo How does this work? When user clicks New Issue button, does the issue automatically get pre-populated with all of this content? If so, that's awesome!

**Comment by MarcelGerber** [comment]
@redmunds Yes, that's it.
See https://github.com/blog/2111-issue-and-pull-request-templates

**Comment by nethip** [comment]
@ficristo :+1:

**Comment by zaggino** [comment]
:+1:

**Comment by ficristo** [comment]
@MarcelGerber you are right about this is only for issues, but Atom as similar wording and I don't think it hurts.

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+2. [Second Step]
+3. [and so on...]
+
+**Expected behavior:** [What you expected to happen]
+
+**Actual behavior:** [What actually happened]
+
+### Versions
+
+Please include the OS and what version of the OS you're running.
```
Comment (MarcelGerber) [inline ISSUE_TEMPLATE.md]: Maybe also include the version of Brackets they're running.

**Comment by ficristo** [comment]
I went ahead and merged. If it misses something or there is to much just go ahead too and fix :smile_cat:

**Pull request merged**